langchain/chains/base.ts

```diff
@@ -76,8 +78,8 @@ export abstract class BaseChain implements ChainInputs {
   /**
    * Call the chain on all inputs in the list
    */
-  apply(inputs: ChainValues[]): ChainValues[] {
-    return inputs.map(this.call);
+  async apply(inputs: ChainValues[]): Promise<ChainValues> {
+    return Promise.all(inputs.map(async (i) => this.call(i)));
```

Comment:
This should be chunked to avoid doing too many requests concurrently and going into rate limits